src/lib/containers/table/SynapseTable.tsx

```diff
@@ -218,18 +218,18 @@ export default class SynapseTable extends React.Component<
     // Make call to resolve entity ids
     if (distinctEntityIds.size > 0) {
       const referenceList: ReferenceList = Array.from(distinctEntityIds).map(
-        id => {
+        (id) => {
```

Comment:
do you have a configuration that adds () ? I think it might be new

Reply:
I'm not sure why this has been getting added lately, my eslint doesn't show this as a preference, Ill look into it more so we avoid these unnecessary diffs, thanks for calling this out.